sweep/sweeper_test.go

```diff
@@ -2307,6 +2327,9 @@ func TestClusterByLockTime(t *testing.T) {
 			input4LockTime2.AssertExpectations(t)
 			input5NoLockTime.AssertExpectations(t)
 			input6NoLockTime.AssertExpectations(t)
+
+			// Assert the mocked methods are called as expeceted.
+			mockFeePref.AssertExpectations(t)
```

Comment:
Could also make this into a defer at the top of the closure. Something I've been doing lately to ensure that the expectations are always asserted, and then if you have multiple mocks in play in the test, the method can handle asserting for them all.

Reply:
Done!

Reply:
Done!